src/lexer_tok.c

```diff
@@ -107,15 +107,16 @@ t_error	ft_partition(t_src *src, t_buf *tmp)
 			ft_add_quoted_str(c, src, tmp);
 		else if ((c == ' ' || c == '\t') && tmp->cur_pos > 0)
 			break ;
-		else if ((c == '\n' || c == '|') && tmp->cur_pos > 0)
+		else if ((c == '\n' || c == '|' || c == '<' || c == '>')
+			&& tmp->cur_pos > 0)
 		{
 			ft_unget_char(src);
```

Comment:
Unget_char could return SUCCESS. Then you could return it, saving you 3 lines :D

Reply:
you ingenious bastard you!